src/sql/src/plan/statement/dml.rs

```diff
@@ -886,6 +936,40 @@ pub fn plan_copy(
             }
             _ => sql_bail!("COPY FROM {} not supported", target),
         },
+        (CopyDirection::To, CopyTarget::Filename(url_expr)) => {
+            scx.require_feature_flag(&vars::ENABLE_COPY_TO_FILENAME)?;
+
+            let from = match relation {
+                CopyRelation::Table { name, columns } => {
+                    if !columns.is_empty() {
+                        // TODO(mouli): Add support for this
+                        sql_bail!("specifying columns for COPY <table_name> TO commands not yet supported; use COPY (SELECT...) TO ... instead");
+                    }
+                    CopyToFrom::Id {
+                        id: *name.item_id(),
+                    }
+                }
+                CopyRelation::Select(stmt) => {
+                    if !stmt.query.order_by.is_empty() {
+                        sql_bail!("ORDER BY is not supported in SELECT query for COPY statements")
+                    }
+                    let PlannedRootQuery {
+                        expr,
+                        finishing,
+                        desc,
+                        ..
```

Comment:
```suggestion
                        scope: _,
```

This will prevent any changes to this struct from getting ignored here, which could be important if the extra fields affect the correctness of the query.

Reply:
ah, good point 👍